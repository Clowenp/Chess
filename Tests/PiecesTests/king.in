k 4 0 6 0 2
k 4 0 6 0 0
k 4 4 6 6 1
k 3 3 2 2 0
k 7 7 8 8 4
