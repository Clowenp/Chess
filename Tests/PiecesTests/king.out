i am a king at [4,0]
my possible moves are: 
< [4,1] >
< [5,1] >
< [5,0] >
< >
< >
< >
< [3,0] >
< [3,1] >
the move to [6,0] is NOT possible
i am a king at [4,0]
my possible moves are: 
< [4,1] >
< [5,1] >
< [5,0] >
< [6,0] >
< >
< >
< >
< [2,0] >
< [3,0] >
< [3,1] >
the move to [6,0] is possible
i am a king at [4,4]
my possible moves are: 
< [4,5] >
< [5,5] >
< [5,4] >
< [5,3] >
< [4,3] >
< [3,3] >
< [3,4] >
< [3,5] >
the move to [6,6] is NOT possible
i am a king at [3,3]
my possible moves are: 
< [3,4] >
< [4,4] >
< [4,3] >
< [5,3] >
< [4,2] >
< [3,2] >
< [2,2] >
< [1,3] >
< [2,3] >
< [2,4] >
the move to [2,2] is possible
i am a king at [7,7]
my possible moves are: 
< >
< >
< >
< >
< [7,6] >
< [6,6] >
< [6,7] >
< >
the move to [8,8] is NOT possible
